ads/adblade.js

```diff
@@ -34,6 +34,11 @@ function addAdiantUnit(hostname, global, data) {
   ins.setAttribute('data-tag-type', 1);
   global.document.getElementById('c').appendChild(ins);
 
+  ins.parentNode.addEventListener(
+    'eventAdbladeRenderStart',
```

Comment:
for better readability, could you indent extra two spaces for this line and the following line, that's what we do if they should originally fall in one line. aka, here we should make them indented 6 spaces here.